docs/user_guides/client_installation/index.md

```diff
@@ -12,7 +12,7 @@ The Hopsworks client library is required to connect to the Hopsworks Feature Sto
 ```
 pip install hopsworks
 ```
-Supported versions of Python: 3.7, 3.8, 3.9, 3.10, 3.11 ([PyPI ↗](https://pypi.org/project/hopsworks/))
+Supported versions of Python: 3.8, 3.9, 3.10, 3.11, 3.12 ([PyPI ↗](https://pypi.org/project/hopsworks/))
```

Comment:
Just a reminder, python 3.7 is supported in hsfs here: https://github.com/logicalclocks/feature-store-api/pull/1153/files#diff-eb8b42d9346d0a5d371facf21a8bfa2d16fb49e213ae7c21f03863accebe0fcfR20